README.md

```diff
@@ -1,30 +1,35 @@
+## Structure
+The structure of the collections follows the structure of Mollie’s public API documentation. There is a request to each endpoint, and each request contains at least one example. The example set aims for showcasing the variety of options an endpoint offers, rather than showing every possibility.
 
-## OAUTH
-If you want to retrieve the access_tokens of another account using OAuth, you can use the endpoints in the Connect folder of the collection that utilises OAuth/access_tokens. Follow these steps:
-1. Start with registering an app in the “partner” account (the one that takes control) in your Mollie Dashboard (‘Developers’ --> ‘Your apps’). 
-2. Place the client id and secret in the authorization settings ‘OAuth API’ folder or the environment variables.
-3. Send the ‘Authorize’ call. This will construct the OAuth URL, which you can fetch from the Postman console.
-4. Make sure you are logged in to the “submerchant” Mollie account (the account that is controlled) and navigate to the URL in your preferred browser.
-5. Connect the accounts and retrieve the ‘code’ parameter from the landing page URL. If you were not logged into Mollie, you can do that here.
-6. Place the copied code in the ‘code’ parameter in the payload of the ‘Generate access tokens’ call in the same collection folder and send it (this may take no longer than 30 seconds. If the code expires, go back to step 4).
-7. The Postman tests will read Mollie’s response and assign the tokens to global variables for further use. You can know use all other endpoints this collection and control the “submerchant” account.
-After one hour, the access_token needs to be refreshed by the ‘Refresh tokens’ endpoint. Simply send it and the variables holding the tokens will be overwritten with the new tokens.
+## Collection variables
+Most endpoint payloads contain variables instead of static values. All variables are initiated at the collection level, and contain dummy data that you can easily adjust. The saved examples display the actual values.
 
-## TESTMODE
-Keep in mind to use the collections with an API key for test mode. The collection based on OAuth/access_tokens uses the test mode query-parameter (GET/DELETE) or payload-parameter (PUT/PATCH) by default.
+## Credentials
+The JSON files contain variables instead of credentials. Postman supports authentication inheritance, so the variables are often called on higher-level objects (collection or folder). Best practice is to create a Postman Environment and initiate all credentials there to prevent credentials from appearing in the collection itself. Besides, you can easily create multiple environments and use them to switch between Mollie accounts or App credentials.
```

Comment:
I would ask Jonas what he means exactly by "so the variables are often called on higher-level objects". This seems like a weird construction to me (speaking as a former developer). Does he mean that the variables reference higher level objects or are they contained in higher level objects?

Reply:
@Jonas4Mollie please clarify